python/orca/src/bigdl/orca/data/shard.py

```diff
@@ -155,6 +155,9 @@ def __init__(self, rdd, transient=False):
         if self.eager:
             self.compute()
         self.type = {}
+        if class_name:
+            self.type['class_name'] = class_name
```

Comment:
where is it used?

Reply:
in many shards operations, we will check if its class_name is `pandas.core.frame.DataFrame`. eg, https://github.com/intel-analytics/BigDL/blob/main/python/orca/src/bigdl/orca/data/shard.py#L536

Reply:
OK; you may want to optimize `_get_class_name` and `_get_schema` (e.g., using one `rdd.map` instead of two).